.circleci/config.yml

```diff
@@ -105,22 +129,54 @@ jobs:
               "${ARNOLD_IMAGE}" \
               ansible --version
 
+  # FIXME: we have deactivated plugins test coverage as the container user is
+  # not allowed to write to /app/.coverage and CircleCI does not allowed to
+  # mount volumes with "docker run" commands.
```

Comment:
you're saying this and then below you're mounting /app/.pytest_cache?

Reply:
It's not a classical volume it's a `tmpfs`. Concerning volumes, I think I cannot use them since we are in "dind" mode. We need to switch to the machine executor at least for this task.

Reply:
So why can't you use a tmpfs for /app/.coverage also?

Reply:
Because it's a file, not a directory.